README.md

```diff
@@ -1 +1,9 @@
 # docs repo
+
+This is the repository for Newport Data Project documentation, such as
+contributing guidelines, participant orientation, and general Project help and best
+practices.
+
+## Contributing Documentation
+To add a new document, fork the repository and create a markdown (\*.md) file
```

Comment:
Consider using an ordered list?  e.g.
1.  Fork the repository
1.  Created a `.md` file
1.  etc

Also, not sure if this is better put into a [_CONTRIBUTING.md_](https://github.com/blog/1184-contributing-guidelines)?

Reply:
The contributing piece might be better in a CONTRIBUTING file, but at the same time is there a risk that it could get confused with the more general contributing documentation for NDP?

We may be able to get around that if we use a directory structure that makes it more clear.  We'll need to settle on a display framework for the web, like jekyll or something else.

Reply:
Well, I think the principal advantage is that in the context of operating within GitHub, each time  time an issue or PR is opened, that user is prompted with a banner to read the CONTRIBUTING.md. 

So as far as getting into people's field of vision, that seems like a pretty good tactic.